doc/source/geotransformations.md

```diff
@@ -0,0 +1,204 @@
+loss of information (value interpolation, propagation of nodata).
+
+For rasters, it can be useful to use {func}`~geoutils.Raster.reproject` in the same CRS simply for re-gridding, 
+for instance when downsampling to a new resolution {attr}`~geoutils.Raster.res`.
+
+```{tip}
+Due to the loss information when re-gridding, it is important to **minimize the number of reprojections during the 
+analysis of rasters** (performing only one, if possible). For the same reason, when comparing vectors and rasters in 
+different CRSs, it is usually **better to reproject the vector with no loss of information, which is the default 
+behaviour of GeoUtils in raster–vector–point interfacing**.
+```
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+:mystnb:
+:  code_prompt_show: "Show the code for opening example files"
+:  code_prompt_hide: "Hide the code for opening example files"
+
+import matplotlib.pyplot as plt
+import geoutils as gu
+rast = gu.Raster(gu.examples.get_path("everest_landsat_b4"))
+rast.set_nodata(0)  # Annoying to have to do this here, should we update it in the example?
+rast2 = gu.Raster(gu.examples.get_path("everest_landsat_b4_cropped"))
+vect = gu.Vector(gu.examples.get_path("everest_rgi_outlines"))
+```
+
+```{code-cell} ipython3
+# Reproject vector to CRS of raster by simply passing the raster
+vect_reproj = vect.reproject(rast)
+# Reproject raster to smaller bounds and different X/Y resolution
+rast_reproj = rast.reproject(
+    res=(rast.res[0] * 2, rast.res[1] / 2),
+    bounds={"left": rast.bounds.left, "bottom": rast.bounds.bottom, 
+            "right": rast.bounds.left + 10000, "top": rast.bounds.bottom + 10000},
+    resampling="cubic")
+```
+
+```{code-cell} ipython3
+:tags: [hide-input]
+:mystnb:
+:  code_prompt_show: "Show the code for plotting the figure"
+:  code_prompt_hide: "Hide the code for plotting the figure"
+
+f, ax = plt.subplots(1, 2)
+ax[0].set_title("Before reprojection")
+rast.plot(ax=ax[0])
+vect.plot(rast, ax=ax[0], ec="k", fc="none")
+ax[1].set_title("After reprojection")
+rast_reproj.plot(ax=ax[1])
+vect_reproj.plot(ax=ax[1], ec="k", fc="none")
+_ = ax[1].set_yticklabels([])
+```
+
+```{note}
+In GeoUtils, `"bilinear"` is the default resampling method. A simple {class}`str` matching the naming of a {class}`rasterio.enums.Resampling` method can be
+passed.
+
+Resampling methods are listed in **[the dedicated section of Rasterio's API](https://rasterio.readthedocs.io/en/latest/api/rasterio.enums.html#rasterio.enums.Resampling)**.
+```
+
+We can also simply pass another raster as reference to reproject to match the same CRS, and re-grid to the same bounds 
+and resolution:
+
+```{code-cell} ipython3
+# Reproject vector to CRS of raster by simply passing the raster
+rast_reproj2 = rast.reproject(rast2)
+```
+
+As shown above, if the rasters have different {ref}`Pixel interpretation<pixel-interpretation>`, GeoUtils 
+raises a warning to ensure this is intended. 
```

Comment:
Probably not easy to fix, but the warning is difficult to see at it shows the path to the file and extends beyond cell view:
<img width="814" alt="image" src="https://github.com/GlacioHack/geoutils/assets/3285905/f9e8c8b7-130b-42de-ad3f-31c59ff6a320">

Reply:
Tried to use a `warn` mode of myst-nb but the output is still the same... Not sure if there is a way around it.